<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtBrownianMotionParametersWidget</class>
 <widget class="QWidget" name="QtBrownianMotionParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QSpinBox" name="box_seed">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>42</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_volatility">
     <property name="toolTip">
      <string>sigma</string>
     </property>
     <property name="text">
      <string>Volatility</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="box_volatility">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_init_x">
     <property name="text">
      <string>Initial value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_t_end">
     <property name="text">
      <string>Final time</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="box_t_end">
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_seed">
     <property name="text">
      <string>RNG seed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="box_init_x">
     <property name="minimum">
      <double>-1000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>box_init_x</tabstop>
  <tabstop>box_t_end</tabstop>
  <tabstop>box_volatility</tabstop>
  <tabstop>box_seed</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
